src/modules/GroupChannelList/context/GroupChannelListProvider.tsx

```diff
@@ -120,33 +154,71 @@ export const GroupChannelListProvider = (props: GroupChannelListProviderProps) =
     },
   });
 
+  useEffect(() => {
```

Comment:
I thought the props of provider and internal state handled inside the provider should be here. Is it right?

Reply:
@git-babel Yeah it is correct